examples/layer-browser/src/app.js

```diff
@@ -292,7 +293,7 @@ export default class App extends PureComponent {
           onLayerHover={this._onHover}
           onLayerClick={this._onClick}
           useDevicePixels={useDevicePixels}
-          debug={false}
+          debug={true}
```

Comment:
This flag also used to create debug `WebGL` context, wondering if overloading is fine or we create a new flag `validateProps` ?

Reply:
I really don't want to use another prop. Ideally I wish this could do what React does (switch between dev and prod build by looking for `NODE_ENV=production`)

Reply:
Opened issue https://github.com/uber/deck.gl/issues/2386